src/blog/new-truffle-release-update-march/index.md

```diff
@@ -0,0 +1,108 @@
+
+### Try out the zero-config mainnet forking feature on ganache.dev
+
+Head to Etherscan and grab the latest block number on mainnet, at the time of this writing, it was 16949139 in decimal, which translates to 0x1029F93 in hex.
+
+On ganache.dev, head to the eth_getBlockByNumber RPC method documentation and update the code sample to look like this:
+
+```javascript
+const options = { fork: { network: "mainnet" } };
+const provider = ganache.provider(options);
+
+const block = await provider.request({
+  method: "eth_getBlockByNumber",
+  params: ["0x1029F93", false],
+});
+
+console.log(block);
+```
+
+As you can see, we’re defining a new provider, and passing in the option that enables mainnet forking and since ganache.dev runs on the latest version of Ganache, we get to have this feature on our browser.
+
+Next, we pass the latest mainnet block number as a parameter to the eth_getBlockByNumber RPC method and finally, we log the output to the console.
+
+Once you run this code, you should get a response that looks like this:
+
+![Image of ganache.dev](./getBlockByNumber.png)
+
+If you study the output, you should be able to see that it matches exactly with what we have on Etherscan, so cool!
+
+![Image of etherscan](./etherscan.png)
+
+Feel free to play around with this amazing interactive documentation and if you find any bugs, do not hesitate to [open an issue](https://github.com/trufflesuite/ganache/issues/new).
+
+## What is Truffle Dashboard Hardhat Plugin
+
+Our focus for Truffle going forward is to improve its interoperability with other development frameworks that are available in the ecosystem. This means that we want you to be able to plug any tool from the [Truffle suite](https://trufflesuite.com/blog/) into your existing workflow - irrespective of what development framework you’re using - and still be able to achieve the optimal result.
+
+This is why we’re excited to introduce the [Truffle Dashboard Hardhat Plugin](https://www.npmjs.com/package/@truffle/dashboard-hardhat-plugin), a plugin that extends the Truffle Dashboard’s functionality to allow HardHat users to decode transaction information including the function signature and the values of any arguments passed.
+
+With this plugin, whenever you run “npx hardhat compile”, the compiled artifacts from your Hardhat project are sent to the Truffle Dashboard, which in turn uses [@truffle/decoder](https://trufflesuite.com/docs/truffle/codec/modules/_truffle_decoder.html) to decode, providing more context before you go on to sign a transaction.
+
+### How to get started with Truffle Dashboard Hardhat Plugin
+
+The first step is to make sure you have access to [Truffle](https://trufflesuite.com/truffle/) either by downloading it on your local machine using “npm install -g truffle” or by fetching and running the Truffle dashboard remotely using “npx truffle dashboard”
```

Comment:
Perhaps "downloading it to your local machine".

Also “npm install -g truffle” in backticks (as above).